frontend/src/utils/helpers/getBoxStyle.ts

```diff
@@ -8,15 +8,17 @@ const VIDEO_TILE_MARGIN = 8;
 /**
  * Gets style object for positioning video tiles given a layout box and incorporating a 12px separation
  * @param {(Box | undefined)} box - A Layout Box for the element
+ * @param {boolean} isScreenShare - Whether the video tile is a screenshare or not
  * @returns {CSSProperties | undefined} - Style object or undefined if Box was undefined
  */
-const getBoxStyle = (box: Box | undefined): CSSProperties | undefined =>
+const getBoxStyle = (box: Box | undefined, isScreenShare?: boolean): CSSProperties | undefined =>
   box && {
     left: box.left,
     top: box.top,
     // We subtract the margins from width and height
-    width: box.width - VIDEO_TILE_MARGIN,
+    width: box.width - VIDEO_TILE_MARGIN - (isScreenShare ? 0 : 6),
```

Comment:
In the non-screenshare publisher and subscriber tiles, the video tile container was peaking out. We didn't see it before because we weren't using `object-fit: contain`, but we were using `object-fit: cover` which blew up the image. That's also why mobile publishers looked so strange - we were making a 16:9 image out of a 9:16 image 🤠.

We use 6 since there aren't fractional pixels, but I think it should be `6.22` as `14.22 / 8` is a `16 / 9` ratio.